__tests__/Unit/Components/standup/standup.test.tsx

```diff
@@ -0,0 +1,59 @@
+import StandUp from '@/pages/standup';
+import { RouterContext } from 'next/dist/shared/lib/router-context';
+import { NextRouter } from 'next/router';
+import { Provider } from 'react-redux';
+import { store } from '@/app/store';
+import {
+    createMockRouter,
+    renderWithRouter,
+} from '@/test_utils/createMockRouter';
+
+describe('Standup', () => {
+    test('renders the missed updates of the user', () => {
+        const { queryByTestId } = renderWithRouter(
+            <Provider store={store()}>
```

Comment:
why is the Standup component outside of the routercontext.provider?
 I don't see it being used here do we need router context ?